crucible/src/Lang/Crucible/Backend/ProofGoals.hs

```diff
@@ -18,6 +18,7 @@ module Lang.Crucible.Backend.ProofGoals
   , traverseGoals, traverseOnlyGoals
   , traverseGoalsWithAssumptions
   , traverseGoalsSeq
+    -- ** Proving goals
 
```

Comment:
There's nothing under this Haddock section header. Is this intentional?

Reply:
Whoops, no, that's an oversight! Thanks